drafts/developer-guide/pages/data-access-layer/module-config.adoc

```diff
@@ -0,0 +1,38 @@
+include::ROOT:partial$variables.adoc[]
+
+= Compose Module configuration.
```

Comment:
Would it be beneficial to add annotated screenshots alongside every step outlined below? This is what we've done before -- https://docs.cortezaproject.org/corteza-docs/2022.3/integrator-guide/authentication/mfa.html